src/llvm-gc-interface-passes.h

```diff
@@ -46,6 +46,7 @@
 #include "llvm-pass-helpers.h"
 #include <map>
 #include <string>
+#include <optional>
```

Comment:
Should this include be in `src/llvm-late-gc-lowering.cpp` rather than here? I don't see uses of `optional` in this file.

Reply:
I was just following the pattern where that pass doesn't include any of it's own headers